filter.py

```diff
@@ -1,51 +1,54 @@
-#--------Helper Functions to be used for sorting and filtering-------#
+"""Helper Functions to be used for sorting and filtering."""
 import datetime
 
 
-# Filters by all of the criteria shown below, returns an array of filtered releases
-def filter(data, state, label, start_date, end_date, datetype):
+def filter_releases(data, state, label, start_date, end_date, datetype):
+  """Filters by all of the criteria shown below, returns an array of filtered releases."""  # pylint: disable=line-too-long
+  # convert variables from unix type to something usable
+  state = int(state)
+  start_date = int(start_date)
+  end_date = int(end_date)
 
-    # convert variables from unix type to something usable
-    state = int(state)
-    start_date = int(start_date)
-    end_date = int(end_date)
+  filtered = []
 
-    filtered = []
+  # Validate end_date value
+  if end_date <= 0:
+    end_date = (datetime.datetime.now()-datetime.datetime(1970, 1, 1)).total_seconds()
 
-    # Validate end_date value
-    if end_date <= 0:
-        end_date = (datetime.datetime.now()-datetime.datetime(1970,1,1)).total_seconds()
+  for item in data.values():
+    if item[datetype] >= start_date and item[datetype] <= end_date:
+      if item['state'] == state or state == 0:
+        if label != 'null':
```

Comment:
null should be replaced by empty string when transforming from JSON to Release object.